### ISSUE TITLE
Error when using file browser "The folder contents could not be displayed"

### ISSUE THREAD

**Issue by tthies0** [flxzt/rnote#344]
**Describe the bug**
An Error message pops up when I try to save or open a file. The message is : 
The folder contents could not be displayed
Operation not supported

**To Reproduce**
1. Go to any action that uses the file browser (like open, save, save as...)

**Expected behavior**
No Error

** Console Output **
.....
(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_new_bytestring: assertion 'string != NULL' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_new_variant: assertion 'value != NULL' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_get_type: assertion 'value != NULL' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_type_is_subtype_of: assertion 'g_variant_type_check (type)' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_builder_add_value: assertion '!GVSB(builder)->expected_type || g_variant_is_of_type (value, GVSB(builder)->expected_type)' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_builder_end: assertion 'GVSB(builder)->offset >= GVSB(builder)->min_items' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_get_type: assertion 'value != NULL' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_type_is_subtype_of: assertion 'g_variant_type_check (type)' failed

(rnote:2): GLib-CRITICAL **: 12:50:58.485: g_variant_builder_add_value: assertion '!GVSB(builder)->expected_type || g_variant_is_of_type (value, GVSB(builder)->expected_type)' failed

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version: 22.04.1 LTS
 - Installation source Flatpak

**Additional context**
I can still save or open the file, the error just goes behind rnote and rnote becomes unresponsive until I close the error message

**Comment by tthies0** [comment]
Even when i give permission to all files using Flatseal the error comes up, and i also cant change anything in the workspace tab. The options there just dont seem to do anything.

**Comment by flxzt** [comment]
Yeah that was wrong, I currently don't know what the cause is. The message appeared a few times for me too, but not anymore.

**Comment by benjamin051000** [comment]
Having same exact issue here. If it helps, I get this bug on my Microsoft Surface Pro 5 running the surface-linux kernel & Fedora 37, but not on my desktop, which is just running Fedora 37 straight.

**Comment by benjamin051000** [comment]
Just ran with `RUST_LOG=rnote=debug flatpak run ...`, no change in error logging on the command line as far as I can tell

**Comment by flxzt** [comment]
I believe this is the following bug: https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/issues/29 , so it  would need to be fixed upstream. I have to look into it further if in the meantime this can be prevented by a check on rnote's side or by switching to a new & different GTK API for picking files
